data/remnant/remnant 2 cognizance.txt

```diff
@@ -1166,14 +1166,14 @@ mission "Remnant: Cognizance 25"
 			commodities Food Clothing Metal Plastic Medical "Heavy Metals"
 			outfitters "Korath Basics" "Korath Exiles"
 			variant
-				"Korath Chaser" 3
-		dialog "You have destroyed the ships that the raider left behind."
+				"'olofez" 3
+		dialog "You have destroyed the ships that the Palavret left behind."
 	on waypoint
 		dialog "You have patrolled all six systems."
 	on complete
 		payment 250000
 		conversation
-			`Back on Ssil Vida, you check in with Gavriil and report the destruction of three Korath chasers that you located. "Three?" They sign quizzically. "That is an odd number. Raiders only carry two at most. I will report this right away. But in the meantime, come take a look at what I have set up." They lead you off toward the spaceport.`
+			`Back on Ssil Vida, you check in with Gavriil and report the destruction of three 'olofezs that you located. "Three?" They sign quizzically. "That is an odd number. Palavrets only carry two at most. I will report this right away. But in the meantime, come take a look at what I have set up." They lead you off toward the spaceport.`
```

Comment:
Is this to say they thought it was a Palavret?

Reply:
@Zitchas has stated this is the Tubfalet.

Reply:
Yes, the Remnant have never encountered a tubfalet before, so they assume it is a palavret.

Reply:
aha, ninja'd